### PR TITLE
Add constant for activityId delimiter

### PR DESCRIPTION
**Pull request by DeepanshuA**
# Description

It simply adds a constant for Activity Id Deliemiter (`::`).

## Issue reference

Please reference the issue this PR will close: #6313 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Review by shubham1172 — APPROVED** [review]

**Review by shubham1172 — APPROVED** [review]

File: utils/utils.go [hunk 1]
```diff
@@ -27,7 +27,8 @@ import (
 )
 
 const (
-	DotDelimiter = "."
+	DotDelimiter          = "."
+	WFActivityIDDelimiter = "::" // Delimiter between workflow id and activity id
```
Comment (JoshVanL) [inline utils/utils.go:31]: I think this const would be better to live in `pkg/runtime/wfengine`, or ideally, in `pkg/runtime/consts` (I am currently in the process of breaking up `runtime.go` where I have also created this package).

**Review by JoshVanL — CHANGES_REQUESTED** [review]

**Comment by daixiang0** [comment]
@DeepanshuA could you update it?

**Comment by mukundansundar** [comment]
@DeepanshuA please fix the conflicts and review comments.

**Comment by mukundansundar** [comment]
@DeepanshuA Please fix conflicts in the PR and address review comments.

**Pull request closed without merging**